bubo/bot_commands.py

```diff
@@ -609,3 +612,15 @@ async def _users(self):
         if not text:
             text = help_strings.HELP_USERS
         await send_text_to_room(self.client, self.room.room_id, text)
+    async def _keys(self):
+        subcommand = self.args[0]
+        if subcommand == "create":
+            if self.args[1] == "permanent":
+                key = create_permanent_key(self.config.pindora_id)
+                await send_text_to_room(self.client, self.room.room_id, f"Your permanent code is {key}")
+            elif self.args[2] == "temp":
+                key = create_new_key(self.config.pindora_id, self.args[2], self.args[3], self.args[4], self.args[5], self.args[6])
```

Comment:
Again, we may not have these parameters.

Also it's best use full words, so `temporary` instead of `temp`.